flang/lib/Parser/parsing.cpp

```diff
@@ -159,8 +159,9 @@ void Parsing::EmitPreprocessedSource(
         // which signifies a comment (directive) in both source forms.
         inDirective = true;
       }
-      if (inDirective && directive.size() < directiveNameLength &&
-          IsLetter(ch)) {
+      bool inDirectiveSentinel =
```

Comment:
Always use braced initialization in the frontend.

Reply:
@klausler Thanks for looking at this!
Changed to braced initialization.